folium/plugins/geocoder.py

```diff
@@ -50,12 +155,24 @@ class Geocoder(JSCSSMixin, MacroElement):
         )
     ]
 
-    def __init__(self, collapsed=False, position="topright", add_marker=True, **kwargs):
+    def __init__(self, collapsed=False, position="topright", add_marker=True,
```

Comment:
to satisify our linter these should all be on their own line. Please also add type hints. For example:

```
def __init__(
        self,
        collapsed: bool = False,
        position: str = "topright",
```

You can run the autoformatter/linter locally with precommit:

- Install precommit https://pre-commit.com/#install
- Enable precommit for this repo: `pre-commit install`
- Run precommit on all files: `pre-commit run --all-files`
  - or of course only on the files you touched

Reply:
Have applied the snake_case, type-hints and ran it through `pre-commit`. Please take a look when you have time.